packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -126,7 +169,7 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
             trace('lca', lca);
             const delegatorAddress = await E(lca).getAddress();
             trace('delegatorAddress', delegatorAddress);
-            const result = await E(lca).executeTx([
+            const [result] = await E(lca).executeTx([
```

Comment:
wild... was there a bug here?

Reply:
The expected return for `delegate()` is `Promise<void>`, so no. In the next pass I can change the return statement of this function.